6/debian9/6.7/license-checksum

```diff
@@ -0,0 +1,2 @@
+7a8086e36c141979221311787c1187f1091018b29239a3b168d90b36bc25f993  LICENSE
```

Comment:
Please remove this file, it's not needed

Reply:
Done

Reply:
Are you sure? Please confirm it.